content/guides/figma/contribute.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: How to contribute
+description: We love contributions and are happy to support you.
+---
+
+import createBranchImg from '../../../src/images/figma/create-branch.png'
+import createBranchDialogImg from '../../../src/images/figma/create-branch-dialog.png'
+import requestReviewImg from '../../../src/images/figma/request-review.png'
+import mergeDialogImg from '../../../src/images/figma/merge-dialog.png'
+import requestDialogImg from '../../../src/images/figma/request-dialog.png'
```

Comment:
We don't usually keep assets like this in the repo. We just generate a URL by adding them to an issue comment.

Reply:
Why is this better? I know we often do this, but it seems somehow "wrong" to me.

Reply:
I think its just a lightweight way of storing images, but I don't know if I have the right answer. I always assumed it was because of perf and maintenance (as in, less to maintain in the repo?).

Reply:
I personally _don't_ think it's better and would prefer keeping them in version control like you're doing here. It's just how we've been handling assets in our docs.

Maybe somebody who's been around longer knows the answer. @colebemis ?

Reply:
As long as we are using static images, I think it's more important to make sure we are cataloging them in Figma in a way that people can easily locate and find the editable versions. That said - I don't have strong feelings about storing them in the repo.

Reply:
Great I will merge and if @colebemis or @jonrohan have feedback we can still optimize the image later.